lib/icedfrisby.js

```diff
@@ -271,8 +218,7 @@ class Frisby {
    * @desc Reset Frisby global and setup options
    */
   reset () {
-    this.current.request = _.cloneDeep(_icedfrisbyDefaults().request)
-    return this
+    throw Error('reset() has been removed.')
```

Comment:
I would say removed in version X+ and give a hint to what they should do or link to this PR

Reply:
Fixed